test/language/expressions/optional-chaining/early-errors-tail-position-template-string.js

```diff
@@ -0,0 +1,21 @@
+/*---
+esid: pending
+info: |
+  template string passed to tail position of optional chain
+description: >
+  Static Semantics: Early Errors
+    OptionalChain:
+      ?.TemplateLiteral
+      OptionalChain TemplateLiteral
+features: [optional-chaining]
+negative:
+  type: SyntaxError
+  phase: parse
+---*/
+
+$DONOTEVALUATE();
+
+const a = {fn() {}};
+
+a?.fn
+  `hello`;
```

Comment:
I was specifically trying to address this section of the spec:

> NOTE This production exists in order to prevent automatic semicolon insertion rules (11.9) to be applied to the following code.

... however, by adding a semi-colon to the end of the expression I was failing to actually exercise this behavior, I've split it into another test.

Reply:
thanks for the clarification